src/nodes/hypertable_modify.h

```diff
@@ -27,6 +27,12 @@ typedef struct HypertableModifyState
 	CustomScanState cscan_state;
 	ModifyTable *mt;
 	List *serveroids;
+	/* list of compressed chunks */
+	List *chunks;
+	/* list of conditions specified in WHERE */
+	List *predicates;
```

Comment:
why do we need to keep this in the state we only need this information once to do the batch decompression

Reply:
Right, removed it.